### ISSUE TITLE
macOS Bug report - does not recognize download

### ISSUE THREAD

**Issue by kerim** [campionfellin/CLOxIssueTracker#18]
**Describe the bug**
I followed the instructions to install and download the Chinese speech pack in the accessibility preference pane, but the app still does not recognize this as installed.

**To Reproduce**
Try to transcribe something using Chinese (Taiwan)

**Expected behavior**
It will transcribe. (It does not, says not installed.)

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version App store version

**Additional context**

**Comment by campionfellin** [comment]
Hey @kerim thanks for opening this! Are you getting the message that looks like this?

```
This locale cannot be used on-device. Either you haven't downloaded the Dictation offline pack for it or it's
actually not available. Using Apple server to transcribe (limited to 1 minute).
```

There's a couple reasons that you may get this, but it basically boils down to you don't have the Dictation offline pack installed. It could be that (a) you just recently enabled it and it takes time to download or (b) Apple does not offer offline Dictation for this language/locale.

Are you able to use Dictation or Siri with this language/locale while off WiFi in other apps lie Apple Notes?

Also, if there is a way we can make this error message more clear in the app, we are open to suggestions!

**Comment by kerim** [comment]
As I said, I followed the instructions to install the pack. This includes making sure it was finished downloading.

**Comment by campionfellin** [comment]
In that case, it seems like you are hitting (b) Apple does not offer online Dictation for this language/locale.

Even though the red error messages comes up, it should still attempt to transcribe your audio online - is that not happening for you? I was able to transcribe an English file using Chinese - Taiwan and online transcription. It wasn't very accurate of course, but it still worked.

If you are still having issues, feel free to attach your audio here and I can try to reproduce the problems on my side!

**Comment by kerim** [comment]
It said it would only do one minute(?) so I didn't bother. I was interested in the app for processing longer files.

**Comment by campionfellin** [comment]
Yes, unfortunately we are limited to what Apple can provide. I would suggest that you try https://clox.ling.washington.edu/ which also has Chinese - Taiwan support :)

**Comment by rsquizz** [comment]
@kerim This is an odd problem. Apple gives the idea that there's a language pack available for Taiwanese Chinese, but when I "install" it on my mac, it takes up no disk space.

If you're transcribing long audio files of Taiwanese Mandarin, and you prefer to use the macOS version over the web version, it might still save you some time to transcribe using the setting for Mainland China. You could then convert the output to traditional Chinese characters with an online tool, if desired. CLOx for macOS has done relatively well transcribing English-accented L2 Mandarin for me.

@campionfellin I'm going to leave this issue open so others are aware that this is still a problem, even though the CLOx dev team probably can't do anything about it right now.
